libs/pruviloj/Pruviloj/Core.idr

```diff
@@ -268,3 +274,37 @@ reflexivity =
                 , NamePart `{reflexivity}
                 , TextPart "is not applicable."
                 ]
+
+||| Attempt to swap the sides of equality in a goal.
+symmetry : Elab ()
+symmetry = do
+    [_,_,_,_,_] <- apply (Var `{{sym}}) [True, True, True, True, False]
+       | _ => fail [TextPart "Failed while applying", NamePart `{symmetry} ]
+    solve
+    attack
```

Comment:
There shouldn't be a trailing `attack` here. The style in Pruviloj is that those who want to use binding tactics are responsible for bracketing their binders in `attack` and `solve` themselves. This makes it so that nobody needs to keep track of the state that other tactics leave the hole in.